runelite-client/src/main/java/net/runelite/client/plugins/kourendlibrary/KourendLibraryOverlay.java

```diff
@@ -216,9 +219,11 @@ public Dimension render(Graphics2D g)
 				.forEach(n ->
 				{
 					Book b = library.getCustomerBook();
+					ItemContainer itemContainer = client.getItemContainer(InventoryID.INVENTORY);
+					boolean hasBookInInventory = itemContainer != null && b != null && Arrays.stream(itemContainer.getItems()).anyMatch(item -> item.getId() == b.getItem());
```

Comment:
neither the book nor the inventory can ever be null

Reply:
Client.java, line 1003
```
	@Nullable
	ItemContainer getItemContainer(InventoryID inventory);
```

ItemContainer can be null from the context given.

Book can't be null indeed in this case, since LibraryCustomer is not null, but I find it a bit unsafe assumption that could cause problems in the future.

Reply:
Yes, every other ItemContainer can be null, but not the inventory.
If you want to keep the null checks, atleast split this onto multiple lines

Reply:
Split them up into multiple lines, like the following?

```
boolean hasBookInInventory = itemContainer != null
    && b != null
    && Arrays.stream(itemContainer.getItems()).anyMatch(item -> item.getId() == b.getItem());
```

Reply:
yes

Reply:
Requested changes are done.